tests/functional/test_rose_stem.py

```diff
@@ -251,234 +252,83 @@ def _inner_fn(rose_stem_opts, verbosity=verbosity):
     yield _inner_fn
 
 
-@pytest.fixture(scope='class')
-def rose_stem_run_really_basic(rose_stem_run_template, setup_stem_repo):
+def test_rose_stem_run_really_basic(rose_stem_run_template, setup_stem_repo):
+    """Check that assorted variables have been exported."""
     rose_stem_opts = {
         'stem_groups': [],
         'stem_sources': [
             str(setup_stem_repo['workingcopy']), "fcm:foo.x_tr@head"
         ],
     }
-    yield rose_stem_run_template(rose_stem_opts)
+    plugin_result = rose_stem_run_template(rose_stem_opts)
```

Comment:
I'm not sure `plugin_result` is the right name for this, any suggestions?

Reply:
* stem_outputs
* stem_variables